hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashPolicyOzone.java

```diff
@@ -284,13 +285,14 @@ protected class Emptier implements Runnable {
           + " minutes, Emptier interval = "
           + (this.emptierInterval / MSECS_PER_MINUTE) + " minutes.");
       executor = new ThreadPoolExecutor(trashEmptierCorePoolSize,
-          trashEmptierCorePoolSize, 1,
-          TimeUnit.SECONDS, new ArrayBlockingQueue<>(1024),
+          trashEmptierCorePoolSize, 1, TimeUnit.SECONDS,
+          new ArrayBlockingQueue<>(1024), new ThreadFactory() {
+            @Override
+            public Thread newThread(Runnable r) {
+              return new Thread(r, threadNamePrefix + "TrashEmptier");
+            }
+          },
```

Comment:
Let's use `ThreadFactoryBuilder` to simplify the code:

`new ThreadFactoryBuilder().setNameFormat(...).build()`

Also, please append `-%d` to the end of the thread name format, since this is a pool with multiple threads.

Reply:
Done!